src/applications/static-pages/tests/application-status.e2e.spec.js

```diff
@@ -0,0 +1,105 @@
+const E2eHelpers = require('../../../platform/testing/e2e/helpers');
+const Timeouts = require('../../../platform/testing/e2e/timeouts.js');
+const Auth = require('../../../platform/testing/e2e/auth');
+
+function testStatus(client, page, url) {
+  client
```

Comment:
Not the most extensive tests, but it makes sure the right button shows up and runs the axe check on the page both with and without the start over modal.